nixos/modules/services/databases/victoriametrics.nix

```diff
@@ -11,6 +11,13 @@ let cfg = config.services.victoriametrics; in
         The listen address for the http interface.
       '';
     };
+    storageLocation = mkOption {
+      default = "/var/lib/victoriametrics";
+      type = types.str;
```

Comment:
Is there a reason against using `types.path` here?

Reply:
Not that I can think of!

Reply:
Fixed